Update image-builder-powershell.md (#105322)
Azure/azure-content: articles/virtual-machines/windows/image-builder-powershell.md

If you choose to use PowerShell locally, this article requires that you install the Azure PowerShell
module and connect to your Azure account by using the [Connect-AzAccount](/powershell/module/az.accounts/connect-azaccount) cmdlet. For more information, see [Install Azure PowerShell](/powershell/azure/install-az-ps).

> Note that some of the steps require cmdlets from the [Az.ImageBuilder](https://www.powershellgallery.com/packages/Az.ImageBuilder) module, which needs to be installed separately by

Reviewer: ```suggestion
Some of the steps require cmdlets from the [Az.ImageBuilder](https://www.powershellgallery.com/packages/Az.ImageBuilder) module. Install separately by using the following command.
```
